filter.js

```diff
@@ -153,19 +161,123 @@ const sentEventClick = (eventName) => {
       </div>`;
     };
 
+    const sortList = (arrayList) => {
+      if (sortSettings) {
+        return arrayList.sort((a, b) => a.cost - b.cost);
+      } else {
+        return arrayList.sort(
+          (a, b) =>
+            moment(a.date, "D.MM.YYYY HH:mm").unix() -
+            moment(b.date, "D.MM.YYYY HH:mm").unix()
+        );
+      }
+    };
+
     const renderElement = (arrayList) => {
+      displayAmountOfEvent(arrayList);
+      sortList(arrayList);
+      let oldEvent = arrayList.filter(
+        (workshop) =>
+          moment(workshop.date, "D.MM.YYYY HH:mm").unix() < moment().unix()
+      );
+      let newEvent = arrayList.filter(
+        (workshop) =>
+          moment(workshop.date, "D.MM.YYYY HH:mm").unix() > moment().unix()
+      );
       let contenerList = ``;
+      let workshopClass = "";
+
       if (!arrayList.length) {
         renderEmpty();
       } else {
-        arrayList.forEach((workshop) => {
+        newEvent.forEach((workshop) => {
```

Comment:
dwa razy iteracja z masą kodu - trochę na wyrost bo to się niczym nie różni - warto zrobić jedną funkcję render z parametrem i wio :)